src/main/kotlin/org/sonar/fakeplugin/toolWindow/RealTimeToolWindowFactory.kt

```diff
@@ -0,0 +1,19 @@
+package org.sonar.fakeplugin.toolWindow
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.Disposer
+import com.intellij.openapi.wm.ToolWindow
+import com.intellij.openapi.wm.ToolWindowFactory
+import com.intellij.ui.content.ContentFactory
+
+class RealTimeToolWindowFactory : ToolWindowFactory {
+
+    override fun createToolWindowContent(project: Project, toolWindow: ToolWindow) {
+        val realTimeToolWindow = RealTimeToolWindow(project)
+        val contentFactory = ContentFactory.SERVICE.getInstance()
+        val content = contentFactory.createContent(realTimeToolWindow.getContent(), "", false)
+        Disposer.register(content, realTimeToolWindow)
```

Comment:
Not sure it's working

Reply:
See https://plugins.jetbrains.com/docs/intellij/tool-windows.html#contents-tabs:

> Use Content.setDisposer() to register associated Disposable